benchmark/compare.js

```diff
@@ -40,6 +40,12 @@ if (benchmarks.length === 0) {
   return;
 }
 
+const cpuCoreSetting = cli.optional.set.find((s) => s.startsWith('CPUCORE='));
+let cpuCore = null;
+if (cpuCoreSetting) {
+  cpuCore = cpuCoreSetting.split('=')[1];
```

Comment:
Can we make an assert here?

Reply:
Also we need to convert this to number, I assume.

Reply:
@anonrig no it can be taskset -c 1-2

Reply:
no it can be `taskset -c 1-2`